### PR TITLE
use http_client() from castor

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: .castor/docker.php [hunk 1]
```diff
@@ -32,9 +34,11 @@ function about(): void
     io()->section('Available URLs for this project:');
     $urls = [variable('root_domain'), ...variable('extra_domains')];
 
-    $payload = @file_get_contents(sprintf('http://%s:8080/api/http/routers', variable('root_domain')));
-    if ($payload) {
-        $routers = json_decode($payload, true);
+    try {
+        $routers = http_client()
+            ->request('GET', 'http://127.0.0.1:8080/api/http/routers')
```
Comment (damienalexandre) [inline .castor/docker.php:39]: it was root_domain before, why this change?

Comment (lyrixx) [reply .castor/docker.php:39]: because it's useless to use the root domain. It's simpler like that, isn't?

Comment (lyrixx) [reply .castor/docker.php]: hmm, I don't know how docker machine works, but I have rollback this change

**Review by pyrech — APPROVED** [review]

**Comment by lyrixx** [comment]
FYI: We need to wait for a castor release

**Comment by pyrech** [comment]
@lyrixx Could you rebase this PR?

**Comment by lyrixx** [comment]
@pyrech Done ✅

**Review by pyrech — APPROVED** [review]
